cobigen/index.asciidoc

```diff
@@ -68,3 +64,11 @@ For example, the following files are generated by CobiGen when using the specifi
 (8) CRUD SpringData Repository: Generates the entity repository (that contains the CRUD operations) in the data access layer.
 - `devonfw/workspaces/main/cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/repo/CustomerRepository.java`{{open}}
 ====
+
+Furthermore, we can change the CustomerEntity afterwards and update our source files simply by running cobigen again.
+[step]
+== Change Entity and run cobigen again
+--
+changeFile("cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/CustomerEntity.java", { "file": "files/Placeholder.java", "placeholder": "private int age;" })
+cobiGenJava("cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/CustomerEntity.java",[1,3,5,6,8])
```

Comment:
At the moment we have to use the adaptTemplates command before we use cobiGenJava. I would insert it at the beginning after the installCobiGen step or just before cobiGenJava.